src/encoded/static/components/cart/add_multiple.js

```diff
@@ -86,7 +86,8 @@ class CartAddAllSearchComponent extends React.Component {
         const { savedCartObj, inProgress } = this.props;
         const cartName = (savedCartObj && Object.keys(savedCartObj).length > 0 ? savedCartObj.name : '');
         return (
-            <React.Fragment>
+            <div className="cart-add-all">
+                {savedCartObj && savedCartObj.name ? <div className="cart-toggle__name">{truncateString(savedCartObj.name, 22)}</div> : null}
```

Comment:
The Jira ticket tells me why 22 is used? But a future code-reader may not know why. Maybe a const with an appropriate name would make it clearer.